requirements.in

```diff
@@ -0,0 +1,3 @@
+-e git+https://github.com/dimagi/commcare-cloud.git@dmr/prep-for-separate-env#egg=commcare-cloud
```

Comment:
I assume we'd version this at some point right?

Reply:
Yeah my idea of the progression is that once that PR is merged, we'll point this to master; and then once we're uploading versioned releases to pypi, we'll change this line to just `commcare-cloud` (or `commcare-cloud[test]`) and pin it in requirements.txt via `pip-compile`.

Could eventually use some revisiting as it's a little funny to have a requirements file here that's pretty parallel to the one in `commcare-cloud`, but that should be too much of an issue.